Services/Store/StoreAutocompleteConfig.php

```diff
@@ -118,6 +119,11 @@ public function getPagesTitle()
         return $this->configManager->getStoreConfig(self::WIZZY_AUTTOCOMPLETE_PAGES_TITLE, $this->storeId);
     }
 
+    public function hasToSyncPages()
+    {
+        return ($this->configManager->getStoreConfig(self::WIZZY_AUTTOCOMPLETE_SYNC_PAGES, $this->storeId));
```

Comment:
Compare here the value with 0 e.g. == 0 
So the function either return `true` or `false`